link2aws.js

```diff
@@ -658,7 +658,7 @@ class ARN {
                 "index": null,
             },
             "kinesis": { // Amazon Kinesis
-                "stream": null,
+                "stream": () => `https://${this.region}.console.aws.amazon.com/kinesis/home?region=${this.region}#/streams/details/${this.resource}/details`,
```

Comment:
Can you add a unit test case to cover this change? You can see examples here: https://github.com/link2aws/link2aws.github.io/blob/master/testcases/aws.json